packages/console/src/pages/TenantSettings/TenantBasicSettings/DeleteModal/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { type TenantInfo } from '@logto/schemas';
+import classNames from 'classnames';
+import { useTranslation, Trans } from 'react-i18next';
+
+import DeleteConfirmModal from '@/components/DeleteConfirmModal';
+import TextLink from '@/components/TextLink';
+import { contactEmailLink } from '@/consts';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  isOpen: boolean;
+  isLoading: boolean;
+  onClose: () => void;
+  onDelete: () => void;
+  profile: Pick<TenantInfo, 'name' | 'tag'>;
+};
+
+function DeleteModal({ isOpen, isLoading, onClose, onDelete, profile }: Props) {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { name, tag } = profile;
+
+  return (
+    <DeleteConfirmModal
+      isOpen={isOpen}
+      isLoading={isLoading}
+      expectedInput={name}
+      inputPlaceholder={name}
+      className={styles.deleteConfirm}
+      title="tenants.delete_modal.title"
+      confirmButtonText="tenants.delete_modal.delete_button"
+      onCancel={onClose}
+      onConfirm={onDelete}
+    >
+      <div className={classNames(styles.description, styles.content)}>
+        <ul>
+          <li>
+            <Trans components={{ span: <span className={styles.bold} /> }}>
+              {t('tenants.delete_modal.description_line1', {
+                name,
+                tag,
+              })}
+            </Trans>
+          </li>
+          <li>
+            <Trans
+              components={{
+                span: <span className={styles.highlight} />,
+                a: <TextLink href={contactEmailLink} target="_blank" />,
+              }}
+            >
+              {t('tenants.delete_modal.description_line2')}
+            </Trans>
+          </li>
+          <li>
+            <Trans components={{ span: <span className={styles.bold} /> }}>
+              {t('tenants.delete_modal.description_line3', { name })}
+            </Trans>
+          </li>
+        </ul>
```

Comment:
nit: I think it's unnecessary to use unordered list here, as it invoves extra CSS styles to customize. Can just use `<div>` or `<p>` instead.

Reply:
Tried with <div>, seems we also need to introduce extra CSS styles to keep proper spacing between paragraphs.